weave

```diff
@@ -28,7 +28,7 @@ usage() {
     echo "weave launch       [-password <password>] [-nickname <nickname>] [-nodiscovery]"
     echo "                       [-iprange <cidr>] <peer> ..."
     echo "weave launch-dns   <cidr>"
-    echo "weave launch-proxy [-H <docker_endpoint>] [--with-dns] [--with-ipam]"
+    echo "weave launch-proxy [-D <docker_endpoint>] [-L <listen_address> ...] [--with-dns] [--with-ipam]"
```

Comment:
Neither `<docker_endpoint>` nor `<listen_address>` are defined.

Also, it's `-H`, not `-L`.

And the format for `-L` is wrong, since presumably it needs to precede every address, and not just be supplied once, for all addresses.